tidy3d/components/grid/mesher.py

```diff
@@ -373,7 +373,13 @@ def structure_steps(
         min_steps = []
         for structure in structures:
             if isinstance(structure, Structure):
-                if isinstance(structure.medium, (PECMedium, Medium2D)):
+                if (
+                    isinstance(structure.medium, AnisotropicMedium)
+                    and structure.medium.is_comp_pec(axis)
+                ) or isinstance(structure.medium, Medium2D):
+                    # for 2d medium, will always ignore even if not PEC;
+                    # later, this will be handled by _grid_corrections_2dmaterials
+                    # in simulation.py
```

Comment:
Looks a missing case when the medium is PECMedium? (Before this PR, it was `if isinstance(structure.medium, (PECMedium, Medium2D)):`)

Reply:
good catch, I guess I deleted this by accident.